src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -85,19 +85,22 @@ public void testCreateFeedbackResponseComment() throws Exception {
         verifyPresentInDatastore(frComment);
 
         ______TS("typical successful case: frComment already exists");
```

Comment:
Describe the behaviour when frComment already exists.

Reply:
okay will do